src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -269,6 +270,10 @@ private function parseDefinition($id, $service, $file)
      */
     protected function loadFile($file)
     {
+        if (!class_exists('Symfony\Component\Yaml\Parser')) {
+            throw new RuntimeException('Unable to load YAML files service definitions as the Symfony Yaml Component is not installed.');
```

Comment:
this may be a semantic config file or a parameter file, not only a service definition file. Just use ``Yaml config files``